tower-watch/src/lib.rs

```diff
@@ -3,6 +3,8 @@ extern crate futures;
 extern crate futures_watch;
 extern crate tower_service;
 
+use std::{fmt, error};
+
```

Comment:
tioli: i personally don't think there's much value in splitting std imports out into another block.

Reply:
Removed the whitespace in 142f0fcfe1c85002999925ba8e99f46637314bd0.